scripts/config/templates/nginx.conf.erb

```diff
@@ -22,7 +22,7 @@ http {
 
   server_tokens off;
 
-  split_clients "${request_uri}_DLP_VERSION" $is_dlp_v2 { 
+  split_clients "${remote_addr}_DLP_VERSION" $is_dlp_v2 { 
```

Comment:
what if it's just a bunch of cloudfront servers hitting nginx?

Reply:
I don't understand your question. A client has to send a request to cloundfront that then forwards to nginx.

Reply:
Updated to user x-forwarded-for which will contain chain of ips including real client ip